orion/pkg/repos/migrations/000006_create_table_sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE sessions (
+	id int unsigned NOT NULL AUTO_INCREMENT,
+	created_at datetime NOT NULL,
+	updated_at datetime NOT NULL,
+	deleted_at datetime,
+	class_id varchar(64) NOT NULL,
```

Comment:
double check that `class_id` varchar length is the same as in the classes table.